backend/app/controllers/api/v1/games_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Api::V1::GamesController < ApplicationController
+  def index
+    if params[:new] == "true"
```

Comment:
Looks like this can be the default, without `?new=true`. If you do think it's cleaner, you can also define it on the router, so you want get an empty response without it